src/Files.App/Helpers/Layout/AdaptiveLayoutHelpers.cs

```diff
@@ -47,38 +46,20 @@ private static Layouts GetAdaptiveLayout(string path, IList<ListedItem> filesAnd
 
 		private static Layouts GetPathLayout(string path)
 		{
-			var iniPath = IO.Path.Combine(path, "desktop.ini");
+			var desktopIni = ContentPageContext.ShellPage!.ShellViewModel.DesktopIni;
```

Comment:
I don't think this change is appropriate, because this method is to get information for a given path, not for the current context.

Reply:
Actually this `string path` is always `WorkindDirectory`.
I removed this argument from all callers.

Reply:
I still think we should use the `path` parameter here, because I don't think static helper methods should be context dependent.
Even if we make this context dependent, `ShellPage` should be null-safe.

Reply:
Those methods inside the helper can be merged into single method as well.

Reply:
I see, then please consider `ShellPage` may be null.

Reply:
Done.